homeassistant/components/media_source/__init__.py

```diff
@@ -103,12 +105,32 @@ async def websocket_browse_media(
     """Browse available media."""
     try:
         media = await async_browse_media(hass, msg.get("media_content_id", ""))
+    except BrowseError as err:
+        connection.send_error(msg["id"], "browse_media_failed", str(err))
+    else:
+        expiration = timedelta(seconds=DEFAULT_EXPIRY_TIME)
+
+        def sign_thumbnail(item: BrowseMedia) -> None:
```

Comment:
let's not do this in favor of the frontend solution.

Reply:
👍🏼 removed.